src/main/java/net/sf/jabref/external/MoveFileAction.java

```diff
@@ -143,23 +144,16 @@ public void actionPerformed(ActionEvent event) {
                     }
                     chosenFile = sb.toString();
                 } else {
-                    chosenFile = FileDialogs.getNewFile(frame, file, Collections.singletonList(extension),
-                            JFileChooser.SAVE_DIALOG, false);
+                    System.out.println("Entry extension " + extension);
+                    chosenFile = new NewFileDialogs(frame, file.getPath()).saveNewFile().toString();
+
+
                 }
-                if (chosenFile == null) {
+                if (chosenFile.isEmpty()) {
                     return; // canceled
                 }
                 newFile = new File(chosenFile);
-                // Check if the file already exists:
```

Comment:
Nope, the overwrite dialog is implemented as part of the JFileChooser (overriden method), so it needs not to be handled everywhere